mrSerious/StackOverflow-lite
# #160540074 deploy application front-end to Heroku

#### What does this PR do?
Setup and deploy the application front-end
#### Description of Task to be completed?
- Include the index.php file
- Include the composer.json file
#### How should this be manually tested?
Open your browser and navigate to [the web app](https://stack-overflow-lite-web.herokuapp.com/)
You should be presented with the site landing page.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#160540074](https://www.pivotaltracker.com/n/projects/2189147/stories/160540074)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
